src/pages/index.js

```diff
@@ -4,25 +4,26 @@ import { Link } from "gatsby"
 import Layout from "../components/layout";
 import Image from "../components/image";
 import SEO from "../components/seo";
-import "../components/home.scss"
+import "../styles/pages/index.scss"
+
 import { Container, Col, Row, Button } from 'react-bootstrap';
 
 const IndexPage = () => (
-  <Layout>
+  <Layout pageClassName="index-page">
     <SEO title="Home" />
 
         <div className="hero mx-auto">
             <Image />
             </div>
 
     <Container className="px-0">
-        <Row className="descriptions justify-content-center">
-          <Col sm={12} md={5} className="px-sm-5 py-sm-4 px-3 py-3">
+        <Row className="descriptions justify-content-center mx-0">
+          <Col xs={11} md={5} className="px-md-5 py-md-4 px-3 py-3">
```

Comment:
I found that the card content fit on mobile, but the shadow went off the screen.